pilot/pkg/networking/core/v1alpha3/cluster.go

```diff
@@ -85,6 +90,10 @@ func (configgen *ConfigGeneratorImpl) BuildClusters(env *model.Environment, prox
 func (configgen *ConfigGeneratorImpl) buildOutboundClusters(env *model.Environment, proxy *model.Proxy, push *model.PushStatus,
 	services []*model.Service) []*v2.Cluster {
 	clusters := make([]*v2.Cluster, 0)
+	serviceAccounts, err := getServiceAccounts(env, proxy)
+	if err != nil {
+		return clusters
```

Comment:
We need to log this error.

Reply:
in getServiceAccounts

Reply:
The point is, getServiceAccounts is called by multiple functions, if we log the error there, sometimes it is hard to trace the calling path, i.e., we want to know the origin of the error.